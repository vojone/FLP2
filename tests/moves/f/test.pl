:- consult("../../../cube").

:- read_cube(C, _), move_f_cw(C, _, _, RC), write_cube(RC).
