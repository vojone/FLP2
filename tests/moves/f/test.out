123
456
443
114 711 222 331
114 822 333 442
114 922 333 443
221
666
666
